<ui version="4.0" >
 <author>Matthias Kretz &lt;kretz@kde.org></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>OutputDeviceChoice</class>
 <widget class="QWidget" name="OutputDeviceChoice" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="label" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Select the device preference to be used for audio output. Applications use the given categories below for their output devices (for example all notification sounds from Kopete, kwin and so on are in the "Notifications" category, while media players use the "Music" and "Video" categories depending on the media). This makes it possible to choose the default device and a fallback seperately for every category.</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>&amp;Music &amp;&amp; Video:</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QComboBox" name="musicCombo" />
        </item>
        <item>
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>if not available fall back on:</string>
          </property>
          <property name="textFormat" >
           <enum>Qt::PlainText</enum>
          </property>
          <property name="buddy" >
           <cstring>musicFallbackCombo</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="musicFallbackCombo" />
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QGroupBox" name="groupBox_4" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>&amp;Communication:</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QComboBox" name="communicationCombo" />
        </item>
        <item>
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>if not available fall back on:</string>
          </property>
          <property name="textFormat" >
           <enum>Qt::PlainText</enum>
          </property>
          <property name="buddy" >
           <cstring>communicationFallbackCombo</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="communicationFallbackCombo" />
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1" >
      <widget class="QGroupBox" name="groupBox" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>&amp;Notifications:</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QComboBox" name="notificationCombo" >
          <property name="sizeAdjustPolicy" >
           <enum>QComboBox::AdjustToContents</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>if not available fall back on:</string>
          </property>
          <property name="textFormat" >
           <enum>Qt::PlainText</enum>
          </property>
          <property name="buddy" >
           <cstring>notificationFallbackCombo</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="notificationFallbackCombo" >
          <property name="sizeAdjustPolicy" >
           <enum>QComboBox::AdjustToContents</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutfunction spacing="KDialog::spacingHint" margin="KDialog::marginHint" />
 <pixmapfunction>Small</pixmapfunction>
 <includes>
  <include location="local" >kdialog.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
